### ISSUE TITLE
A potential UI improvement

### ISSUE THREAD

**Issue by furins** [AtomLinter/Linter#668]
**TL;DR**: may be possible to place the warnings/errors list above the find&replace area (or above any other UI element sticked to the bottom bar)? It seems more natural to me to have the components that may grow/shrink/collapse more frequently on the top, and the fixed-height ones to the bottom.

<hr>

This is a really small and low-priority enhancement, I know, but I'm asking just in case it could be trivial to fix. I'm trying to explain what happens with this screen capture:

![find_order](https://cloud.githubusercontent.com/assets/722704/8416933/66b96224-1ea9-11e5-8bd3-8ce91c33c147.gif)

Please observe the pointer: I have to move it up and down every time I'd like to replace an occurrence of text. 

In the given example, when I'm replacing quotes using the find&replace dialog the error counter grows and shrinks because of unterminated strings and this displaces also the "replace" button. I've constantly to look at "where the button is".

there are many workarounds, here are just some of them...

1. to press enter when in the "replace" text input (=use keyboards shortcuts)
2. to disable "lint on fly"
3. to toggle the lint error panel off

...but I was wondering if the linter messages may be placed above other elements of the bottom bar to avoid this "jumping effect". The reason is that, in my opinion, if an element changes its height frequently, it should be closer to the "free side" of the bar -> the upper part of a bottom bar, the left side on a right sidebar... 

This is not an issue, definitely, and may be more an atom-related enhancement than a linter one. I'm just throwing my two cents :smiley: to know your opinion about that. 

thanks in advance
Stefano

**Comment by Eschon** [comment]
:+1: 
This can be quite annoying. The workarounds work but I don't want to have to go change my settings every time I want to do some find & replace.

**Comment by steelbrain** [comment]
Placing linter bar above other bars doesn't really sound like a solution, if you guys have one that works in real life. We would love to hear it.

**Comment by furins** [comment]
@steelbrain Why it may not be a solution? Is it not viable? I'm not talking about changing the z-index (=overlapping bars) when I say "above" and "below", I'm referring to the order the bars will appear from the bottom (from the bottom of the screen find&replace should be the first, linter the second in my example). This would definitely solve the issue I've mentioned, I believe. If not, may you provide a counter-example so I can imagine an alternative approach?

my idea is something like this "pseudocode" (using jquery since I've not idea of the Atom internals)

    $('#linter-panel').closest('atom-panel').insertAfter($('.find-and-replace').closest('atom-panel'));

I'm sorry if I've been unclear, unfortunately English is not my native language.

**Comment by furins** [comment]
I've created a pull request that solves the issue using an optional parameter of `atom.workspace.addBottomPanel`, in accordance with atom documentation. Hope it helps!
